README.md

```diff
@@ -0,0 +1,88 @@
+# README
+
+## Introduction
+
+This script is an example implementation of a shielded transfer using the Light Protocol and **Solana** blockchain. It demonstrates how to initialize a **Solana** wallet, request an airdrop of SOL tokens, set up a test relayer, perform a shield operation, transfer tokens to a recipient, and retrieve transaction information.
+
+## Prerequisites
+
+Before running this script, ensure that you have the following prerequisites:
+
+1. **Node**.js and **npm** installed on your machine.
+2. Access to a **Solana** node running locally.
+3. A **Solana** wallet file (JSON format) located at `~/.config/solana/id.json`. If the file doesn't exist, the script will generate a new wallet for testing purposes.
+
+## Installation
+
+1. Clone the repository or create a new project directory.
+2. Open a terminal and navigate to the project directory.
+3. Run the following command to install the required dependencies:
+
+```shell
+npm install
+```
+
+or
+
+```shell
+yarn install
+```
+
+If you want to start your project from scratch, you can add dependencies by running:
+
+```shell
+npm install @lightprotocol/zk.js @coral-xyz/anchor
+```
+
+or
+
+```shell
+yarn add @lightprotocol/zk.js @coral-xyz/anchor
+```
+
+## Configuration
+
+Before running the script, you may need to modify the following configuration options:
+
+- `SOLANA_PORT` (default: `"8899"`): The port on which the local **Solana** node is running. If your node is running on a different port, update this value accordingly.
+- `process.env.ANCHOR_WALLET`: The path to the **Solana** wallet file. By default, it is set to `~/.config/solana/id.json`. If your wallet file is located elsewhere, modify this value accordingly.
+- `LOOK_UP_TABLE`: The lookup table used for the test relayer. Ensure it is correct for your use case.
```

Comment:
if the provided lookup table doesn't exist a new one will be created

Reply:
✅